invoice/src/main/java/org/killbill/billing/invoice/tree/NodeInterval.java

```diff
@@ -223,7 +225,8 @@ private void adjustRightMostChildSibling(final NodeInterval curNode) {
      * @param callback   custom logic to decide if a given node is a match
      * @return the found node or null if there is nothing.
      */
-    public NodeInterval findNode(final LocalDate targetDate, final SearchCallback callback) {
+    @VisibleForTesting
+    NodeInterval findNode(final LocalDate targetDate, final SearchCallback callback) {
```

Comment:
Can't we just delete it? It seems to be used only in `testSearch`, which I believe tests it.

Reply:
See https://github.com/killbill/killbill/pull/1493/commits/0509ee20a4f8cd2ea83910d6ea4f3a566f970548